docs/binding-path.md

```diff
@@ -0,0 +1,125 @@
+# Custom Containers Path and Secret Path
```

Comment:
this file exist on master and should not be part of this PR. Please rebase your PR from master, do not merge master into your branch.